okhttp-clients/src/main/java/com/palantir/conjure/java/okhttp/RemotingOkHttpCall.java

```diff
@@ -151,7 +151,7 @@ private static ResponseBody buffer(MediaType mediaType, byte[] bodyBytes) {
 
     @Override
     public void enqueue(Callback callback) {
-        DeferredTracer tracer = new DeferredTracer();
+        DeferredTracer tracer = new DeferredTracer("OkHttp: acquire-limiter");
```

Comment:
I don't think this name is accurate as you need to open span now and close it in the success while this only make sure that there's same trace around retries

Reply:
What alternative name would you suggest?